docs/api/rpc-experimental.md

```diff
@@ -7,81 +7,18 @@ sidebar_label: Experimental Methods
 <blockquote class="warning">
 <strong>heads up</strong><br><br>
 
-Experimental API methods are feature-complete, but we do not guarantee them to be backward-compatible or exist forever. For example, we are going to remove `EXPERIMENTAL_genesis_records` API soon and `EXPERIMENTAL_genesis_config` does not have a stable structure (we used to change it quite frequently); we are committed to never do such changes for the non-EXPERIMENTAL endpoints. 
+Experimental API methods are feature-complete, but we do not guarantee them to be backward-compatible or exist forever. 
 
-EXPERIMENTAL endpoints are useful for API evaluation on the real network. While the API is in EXPERIMENTAL state we collect the feedback and can iterate on the input/output structure.
+These endpoints are useful for API evaluation on the real network. While the API is in EXPERIMENTAL state, we collect feedback and can iterate on the input/output structure.
```

Comment:
I don't understand why you move most of the experimental methods out to another file but still keep the changes endpoint here. Seems rather inconsistent to me

Reply:
In slack, you told me to skip the `changes` endpoint for now. I agree that it appears incomplete this way. Should I just work on incorporating this endpoint into the new refactored `rpc.md` now?

Reply:
I think you misunderstood me. I meant that you don't have to try every combination of things to generate examples. I did not mean that you should ignore it.

Reply:
Understood, will work on this now.